### DISCUSSION TITLE
Klick'r 3.1.1

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#568] [Announcements]
## Bugfixes

* Fix import of some scenario generated with versions 3.0.x (https://github.com/Nain57/Smart-AutoClicker/issues/561)
* Fix a possible crash with unusual user action Intents
* Fix a crash when changing the state of a trigger event appearing after in the list
* Fix a possible crash when loosing the media projection permission 

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/3.1.1'>Klick'r 3.1.1</a>.</em>

**Comment by TheoBoy1** [comment]
how do i download

**Reply by Nain57** [reply]
Release note trigger the build of the version. As soon as the build is finished, it is added automatically to the release note.

**Comment by M1nt007** [comment]
My auto clicker just stops after a little bit and I am unable to press pause or play the action again. Is there any solution because the don't kill my app website doesn't show anything
